extensions/amp-iframely/validator-amp-iframely.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-iframely
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-iframely"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-iframely>
+  html_format: AMP
+  tag_name: "AMP-IFRAMELY"
+  requires_extension: "amp-iframely"
+  # either data-id
+  attrs: {  # iframely generated ID of a widget
+    name: "data-id"
+    mandatory_oneof: "['data-id', 'data-url']"
+  }
+  attrs: { # iframe url to build widget from
+    name: "data-url"
+    mandatory_oneof: "['data-id', 'data-url']"
+  }
+  attrs: { name: "data-key" } # iframely key hash
+
+  # optional attributes
+  attrs: { name: "data-img" value: "" }
+  attrs: {
+    name: "data-border" value: ""
+    value_regex: "(\\d+)"
+  }
+  attrs: {
+    name: "data-domain"
+    value_regex: "^((?:[^\\.\\/]+\\.)?iframe\\.ly|if\\-cdn\\.com|iframely\\.net|oembed\\.vice\\.com|iframe\\.nbcnews\\.com)$"
+  }
+  attrs: { name: "resizable" value: "" }
+  # data-* attributes are converted into options
+  # for building iframely widget from url
+  attr_lists: "extended-amp-global"
+  spec_url: "https://amp.dev/documentation/components/amp-iframely"
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: NODISPLAY
+    supported_layouts: RESPONSIVE
+    supported_layouts: INTRISTIC
+    supported_layouts: CONTAINER
```

Comment:
- Remove `CONTAINER`, this element can't be one.
- It's `INTRINSIC`.

```suggestion
    supported_layouts: INTRINSIC
```

You also need to [create validator tests](https://github.com/ampproject/amphtml/blob/main/validator/README.md).

Reply:
Remove `NODISPLAY` as well.